spk/fuse-exfat/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = fuse-exfat
```

Comment:
I think you can chose the one you prefer as long as you stick to it and are consistent.